### ISSUE TITLE
mouse cursror issues on websites & forms

### ISSUE THREAD

**Issue by ghost** [QupZilla/qupzilla#1498]
I'm using 1.8.2 with QT 5.3.2, webkit 538.1.

I noticed on Github here, sometimes the mouse cursor doesn't appear, and when using the left & right keyboard arrows to move it around, it creates some sort of artifacts on the page, like multipile lines; 

||||||||||||||||||

When the cursor was at the end of the sentence, and then I used the left key arrow to move the cursor to the left it caused this effect...

Basically all I can say is the cursor is a bit erratic in behaviour, sometimes working, sometimes not...
